tdrs-backend/tdpservice/users/test/test_models.py

```diff
@@ -111,4 +112,4 @@ def test_user_without_fra_access(client, data_analyst):
     datafile.save()
 
     response = client.get(f"/admin/data_files/datafile/{datafile.id}/change/")
-    assert response.status_code == 302
```

Comment:
this was wrong (not a change in this PR), not sure why we were getting 302 since data analyst is supposed to have access to datafiles page